exasol/slc/internal/tasks/test/test_runner_db_test_base_task.py

```diff
@@ -236,42 +238,38 @@ def get_result_status(status):
                     break
         return result_status
 
-    def get_test_folders(self, test_config):
-        test_folders = []
-        if test_config["test_folders"] != "":
-            test_folders = test_config["test_folders"].split(" ")
-        if self.tests_specified_in_parameters():
-            test_folders = self.test_folders
-        return test_folders
-
-    def tests_specified_in_parameters(self):
-        return (
-            len(self.generic_language_tests) != 0
-            or len(self.test_folders) != 0
-            or len(self.test_files) != 0
-        )
 
-    def get_generic_language_tests(self, test_config):
-        generic_language_tests = []
-        if test_config["generic_language_tests"] != "":
-            generic_language_tests = test_config["generic_language_tests"].split(" ")
-        if self.tests_specified_in_parameters():
-            generic_language_tests = self.generic_language_tests
```

Comment:
do not remove this:
With this check you can still override the ci.json test specification with CLI arguments. 
This is very useful when you need to run a specific test locally.